packages/editor/src/components/parts/output/OutputPart.tsx

```diff
@@ -8,22 +8,22 @@ import { MappingPart, PathCollapsible, ValidationFieldset } from '../common';
 import type { BrowserType } from '../../../components/browser';
 import useMaximizedCodeEditor from '../../browser/useMaximizedCodeEditor';
 
-export function useOutputPart(options?: { hideCode?: boolean; additionalBrowsers?: BrowserType[] }): PartProps {
+export function useOutputPart(options?: { enableSudo?: boolean; additionalBrowsers?: BrowserType[] }): PartProps {
   const { config, defaultConfig, initConfig, resetOutput } = useOutputData();
-  const compareData = (data: OutputData) => [data.output.map, options?.hideCode ? '' : data.output.code];
-  const validations = useValidations(options?.hideCode ? ['output', 'map'] : ['output']);
+  const compareData = (data: OutputData) => [data];
+  const validations = useValidations(['output', 'map']);
   const state = usePartState(compareData(defaultConfig), compareData(config), validations);
   const dirty = usePartDirty(compareData(initConfig), compareData(config));
   return {
-    name: 'Output',
+    name: 'Output Data',
     state,
-    reset: { dirty, action: () => resetOutput(!options?.hideCode) },
-    content: <OutputPart showCode={!options?.hideCode} additionalBrowsers={options?.additionalBrowsers} />
+    reset: { dirty, action: () => resetOutput(options?.enableSudo) },
+    content: <OutputPart enableSudo={options?.enableSudo} additionalBrowsers={options?.additionalBrowsers} />
   };
 }
 
-const OutputPart = (props: { showCode?: boolean; additionalBrowsers?: BrowserType[] }) => {
-  const { config, defaultConfig, update } = useOutputData();
+const OutputPart = (props: { enableSudo?: boolean; additionalBrowsers?: BrowserType[] }) => {
```

Comment:
I would still call it 'showSudo'. Enable means more for me, that it is here per default but disabled...